draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1595,6 +1595,13 @@ recipient to target other hosts in the network.  Therefore,
 implementations should consider heuristics at both the
 sender and receiver to reduce the impact of this.
 
+As described in {{mps}}, a Maximum Packet Size (MPS) is maintained for a MP-DCCP connection.
+If MP-DCCP uses a minimum MPS across all paths,
```

Comment:
If -- As

It is specified in the MPS section that MP-DCCP uses a minimum MPS accross all paths, and I do not see how using a per path value would work, so "if" does not seem appropriate.

Reply:
I'm not sure if this rather could be interpreted as a recommendation. Copied from the section:
`The DCCP application interface SHOULD allow the application to discover the current MPS. This reflects the current supported largest size for the data stream that can be used across the set of all active MP-DCCP subflows.`

We use SHOULD to enable the feature of minimum MPS, which is not a MUST and therefore up to the implementer. At least that is my interpretation.

What do you think, @abrunstrom?

Reply:
Thank you. I adopted your suggestion in https://github.com/tsvwg/ietf-multipath-dccp/pull/272/commits/14e95972c2b158d2930ade3991dd5bb51d00b782